packages/perseus-editor/src/widgets/interactive-graph-editor/start-coords/util.ts

```diff
@@ -146,17 +147,13 @@ export const getQuadraticEquation = (startCoords: [Coord, Coord, Coord]) => {
     );
 };
 
-const findAngle = (point1: Coord, point2: Coord) => {
-    const x = point1[0] - point2[0];
-    const y = point1[1] - point2[1];
-
-    return (180 + (Math.atan2(-y, -x) * 180) / Math.PI + 360) % 360;
-};
-
-export const getAngleEquation = (startCoords: [Coord, Coord, Coord]) => {
-    const [point1, vertex, point2] = startCoords;
-
-    const angle = findAngle(point2, vertex) - findAngle(point1, vertex);
+export const getAngleEquation = (
+    startCoords: [Coord, Coord, Coord],
+    allowReflexAngles?: boolean,
+) => {
+    const vertex = startCoords[1];
+    const allowReflex = allowReflexAngles || false;
+    const angle = getClockwiseAngle(startCoords, allowReflex);
```

Comment:
also chaining might be helpful here, although idk how much that would improve readability 
```diff
- const angle = getClockwiseAngle(startCoords, allowReflex);
- const roundedAngle = angle.toFixed(0);
+ const roundedAngle = getClockwiseAngle(startCoords, allowReflex).toFixed(0);
```